masq_lib/src/messages.rs

```diff
@@ -846,6 +846,43 @@ pub struct UiWalletAddressesResponse {
 }
 conversation_message!(UiWalletAddressesResponse, "walletAddresses");
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct CountryCodes {
+    #[serde(rename = "countryCodes")]
+    pub country_codes: Vec<String>,
+    #[serde(rename = "priority")]
+    pub priority: usize,
+}
+
+impl From<(String, usize)> for CountryCodes {
```

Comment:
A couple things.
* Bert wants you to deconstruct the tuple and give names to the elements to make reading easier.
* I don't like this `From` implementation here in `messages.rs`, because it leaks syntax from `masq` into `masq_lib` where it doesn't belong. I think this `impl` should be in `masq` to confine that knowledge there.

Reply:
I can not move this impl to exit_location_command in masq, it complains about: Only traits defined in the current crate can be implemented for arbitrary types